### PR TITLE
Formainpr

### PR DESCRIPTION
**Pull request by patrick-tripp**
Consolidated multiple branches into one:
cora
CORA_test
cora-test
lohindast

Code and script cleanups and fixes.
Deployment, CORA Adcirc, and Liveocean tested.

### CODE REVIEW

**Review by Michael-Lalime — COMMENTED** [review]
I'm in favor of merging these branches and have no problem with this. Does anyone else have other comments?

**Comment by Michael-Lalime** [comment]
@ZacharyWills @mykelalvis can you take a look at this PR and provide thoughts/approval?

File: cloudflow/cluster/configs/NOS/nos.cora.cfg [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+"platform"  : "AWS",
+"region"    : "us-east-2",
+"nodeType"  : "hpc6a.48xlarge",
+"nodeCount" : 2,
+"tags"      : [ 
+                { "Key": "Name", "Value": "IOOS-cloud-sandbox" },
+                { "Key": "Project", "Value": "IOOS-cloud-sandbox" },
+                { "Key": "NAME", "Value": "liveocean-fcst" }
```
Comment (jonmjoyce) [inline cloudflow/cluster/configs/NOS/nos.cora.cfg:9]: Is using the "Name" and "NAME" keys intentional? Recommend renaming one to `forecast_name` or something.

**Review by jonmjoyce — COMMENTED** [review]
I went through the code and didn't see much for potential merge errors. Overall cleanups look good.

File: cloudflow/cluster/configs/NOS/nos.cora.cfg (hunk 1 above)
Comment (patrick-tripp) [reply cloudflow/cluster/configs/NOS/nos.cora.cfg:9]: It was intentional, AWS console is case sensitive and we were using the NAME tag when I first started this. It is not needed. Users can specify any tags they want or need to use. "Name" is a standard console tag. If a user is running multiple jobs it helps to be more specific: See: https://github.com/ioos/Cloud-Sandbox/blob/formainpr/cloudflow/cluster/configs/RPS/test.cora.cfg for example.

Comment (patrick-tripp) [reply cloudflow/cluster/configs/NOS/nos.cora.cfg]: I removed the NAME tag everywhere.

**Review by jonmjoyce — APPROVED** [review]

**Pull request merged**